packages/components/modules/navigations/Header/AccountMenu/AccountPopover/index.tsx

```diff
@@ -1,81 +1,135 @@
 'use client'
 
-import { FC } from 'react'
+import { FC, Fragment } from 'react'
 
 import { User as BaseUser, useJWTUser, useLogout } from '@baseapp-frontend/authentication'
-import { ClickableAvatar, Popover, usePopover } from '@baseapp-frontend/design-system'
-import { JWTContent, joinWithSeparator } from '@baseapp-frontend/utils'
+import { Popover, usePopover } from '@baseapp-frontend/design-system'
+import { JWTContent } from '@baseapp-frontend/utils'
 
+import { ButtonBase, List } from '@mui/material'
 import Box from '@mui/material/Box'
 import Divider from '@mui/material/Divider'
 import MenuItem from '@mui/material/MenuItem'
 import Stack from '@mui/material/Stack'
-import Typography from '@mui/material/Typography'
 
-import { AccountPopoverProps } from './types'
+import AccountAvatar from '../AccountAvatar'
+import CurrentUserPlaceholder from '../CurrentUserPlaceholder'
+import { PopoverStyles as DefaultPopoverStyles } from './styled'
+import { AccountPopoverProps, ComponentItems } from './types'
 
 const AccountPopover: FC<AccountPopoverProps> = ({
+  accountAvatarUrl,
+  onCloseCallback,
   menuItems = [],
+  PopoverStyles = {},
+  accountInfoSectionController = { show: true, items: [] },
+  menuItemsSectionController = { show: true, items: [] },
+  menuActionsSectionController = { show: true, items: [] },
+  extraSectionControllersList = [],
   logoutButtonLabel = 'Logout',
+  disableCurrentUserPlaceholder = false,
   hideLogoutButton = false,
 }) => {
   const { user } = useJWTUser<BaseUser & JWTContent>()
   const { logout } = useLogout()
+
   const popover = usePopover()
 
-  const handleLogout = async () => {
+  const handleOnClose = () => {
+    if (onCloseCallback) {
+      onCloseCallback()
```

Comment:
can avoid the if statement by doing
```suggestion
      onCloseCallback?.()
```